lib/pulselistener.js

```diff
@@ -65,12 +83,21 @@ var PulseConnection = function(options) {
     namespace:          options.username || ''
   });
 
-  if (!options.connectionString) {
-    options.connectionString = buildPulseConnectionString(options);
-  } else {
+  this.isTaskclusterPulse = false;
```

Comment:
hmm... I somewhat regard that as private... since it shouldn't be read by someone using this code...

Perhaps it would be better to make a `queuePrefix` property instead? Similar to how we talked about it should work in `pulse-publisher`...